output/t30-operator.out

```diff
@@ -0,0 +1,41 @@
+CeedVector length 40
+  0.0101229
+  0.0222381
+  0.0313707
+  0.0362684
+  0.0362684
+  0.0313707
+  0.0222381
+  0.0101229
```

Comment:
Can we calculate what entries should be here, such that the comparison could be added to the test source instead of comparing lists of numbers?